third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -138,25 +138,50 @@ output "cluster_ca_certificate" {
 ## Argument Reference
 
 * `name` - (Required) The name of the cluster, unique within the project and
-    zone.
+location.
 
 - - -
 
-* `zone` - (Optional) The zone that the master and the number of nodes specified
-    in `initial_node_count` should be created in. Only one of `zone` and `region`
-    may be set. If neither zone nor region are set, the provider zone is used.
-
-* `region` (Optional)
-    The region to create the cluster in for
-    [Regional Clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/multi-zone-and-regional-clusters#regional).
-    In a Regional Cluster, the number of nodes specified in `initial_node_count` is
-    created in each of three zones of the region (this can be changed by setting
-     `additional_zones`).
-
-* `additional_zones` - (Optional) The list of additional Google Compute Engine
-    locations in which the cluster's nodes should be located. If additional zones are
-    configured, the number of nodes specified in `initial_node_count` is created in
-    all specified zones.
+* `location` - (Optional) The location (region or zone) in which the cluster
+master will be created, as well as the default node location. If you specify a
+zone (such as `us-central1-a`), the cluster will be a zonal cluster with a
+single cluster master. If you specify a region (such as `us-west1`), the
+cluster will be a regional cluster with multiple masters spread across zones in
+the region, and with default node locations in those zones as well.
+
+* `zone` - (Optional) The zone that the cluster master and nodes should be
+created in. If specified, this cluster will be a zonal cluster. `zone` has been
+deprecated in favour of `location`.
```

Comment:
this is maybe just a note about how we do it in our docs in general, but should we emphasize these are deprecated more? Like, `(Optional, Deprecated)` maybe?

Reply:
Done